content/sensu-go/5.0/guides/create-read-only-user.md

```diff
@@ -11,49 +11,54 @@ menu:
 
-## Why use RBAC
+## Why use RBAC?
 RBAC allows you to exercise fine-grained control over how Sensu users interact 
 with Sensu resources. Using RBAC rules, you can easily achieve **multitenancy** 
-so users only have access to resources within their own organization and environment. 
+so different projects and teams can share a Sensu instance. 
 
-## How to create a Read Only user
-The initial installation of Sensu includes a `default` environment, included in the `default` organization, and an `admin` user with full permissions to create, modify, or delete resouces within Sensu. Using `sensuctl`, you can create new roles for users that give as much or as little access as you see fit. The purpose of this guide is to help you create a new user with read-only access to resources within the `default` organization and environment.
+## How to create a read-only user
+By default, Sensu includes a `default` namespace and an `admin` user with full permissions to create, modify, and delete resources within Sensu.
+Using sensuctl configured as the [default `admin` user][2], you can create new roles for users that give as much or as little access as you see fit.
 
-Sensu creates a default read-only role on startup, and you can use that role when creating a read-only user.
+In this section, you'll create a user with read-only access to resources within the `default` namespace.
+Sensu includes a default read-only role called `view` that you can use to create a read-only user.
 
-Create a new user, and apply the read-only role.
+1. Create a user with the username `alice`:
 {{< highlight shell >}}
-$ sensuctl user create 'read-only-user'  --password 'password' --roles 'read-only'
+sensuctl user create alice --password 'password'
 {{< /highlight >}}
 
-You may also apply the read-only role to an existing user account. Users can
-have multiple roles spanning multiple organizations, each with their own set of
-permissions.
+2. Create a `read-only-user` role binding to assign the `view` role to the `alice` user:
+{{< highlight shell >}}
+sensuctl role-binding create read-only-user --role=view --user=alice
+{{< /highlight >}}
+
+## How to create an event-reader user
+Now let's say you want to create a user that has read-only access to only events within the `default` namespace.
+Since this user needs different permissions from those provided by the default `view` role, you'll need to create a role before creating the user and role binding.
 
-Using sensuctl, you can verify that the role and user have the correct permissions.
+1. Create an `event-reader` role with `get` permissions for `events` within the `default` namespace:
 {{< highlight shell >}}
-$ sensuctl role list-rules read-only
-  Type     Org.      Env.    Permissions
- ────── ───────── ───────── ─────────────
-  *      default    default     read
+sensuctl role create event-reader --get --namespace default --type events
```

Comment:
The role command flags have changed a bit:

```
$ sensuctl role create -h
create a new Role with a single rule

Usage:	sensuctl role create [NAME] --verb=VERBS --resource=RESOURCES [--resource-name=RESOURCE_NAMES] [flags]

Flags:
  -h, --help                    help for create
  -r, --resource strings        resources that the rule applies to
  -n, --resource-name strings   optional resource names that the rule applies to
  -v, --verb strings            verbs that apply to the resources contained in the rule

Global Flags:
      --api-url string      host URL of Sensu installation
      --cache-dir string    path to directory containing cache & temporary files (default "/Users/splourde/Library/Caches/sensu/sensuctl")
      --config-dir string   path to directory containing configuration files (default "/Users/splourde/.config/sensu/sensuctl")
      --namespace string    namespace in which we perform actions (default "default")
```

The equivalent now would look like this: `sensuctl role create event-reader --verb get --verb list --resource events --namespace default` (you could also do `--verb get,list`)